+adi/+AD4858/Rx.m

```diff
@@ -0,0 +1,162 @@
+classdef Rx < adi.common.Rx & matlabshared.libiio.base & adi.common.Attribute
+    % AD4858 Precision ADC Class
+    % adi.AD4858.Rx Receives data from the AD4858 ADC
+    %   The adi.AD4858.Rx System object is a signal source that can receive
+    %   data from the AD4858.
+    %
+    %   rx = adi.AD4858.Rx;
+    %   rx = adi.AD4858.Rx('uri','192.168.2.1');
+    %
+    %   <a href="https://www.analog.com/media/en/technical-documentation/data-sheets/ad4858.pdf">AD4858 Datasheet</a>
+
+    properties (Nontunable)
+        % SampleRate Sample Rate
+        %   Baseband sampling rate in Hz, specified as a scalar
+        %   in samples per second.
+        SampleRate = '500000'
+        SamplesPerFrame = 1024
+    end
+
+    properties (Hidden)
+        % Number of frames or buffers of data to capture
+        FrameCount = 1
+    end
+
+    % Channel names
+    properties (Nontunable, Hidden, Constant)
+        channel_names = { ...
+                         'voltage0', 'voltage1', 'voltage2', 'voltage3', ...
+                         'voltage4', 'voltage5', 'voltage6', 'voltage7'}
+    end
+
+    % isOutput
+    properties (Hidden, Nontunable, Access = protected)
+        isOutput = false
+    end
+
+    properties (Nontunable, Hidden)
+        Timeout = Inf
+        kernelBuffersCount = 2
+        dataTypeStr = 'int32'
+        phyDevName = 'ad4858'
+        devName = 'ad4858'
+    end
+
+    properties (Nontunable, Hidden, Constant)
+        Type = 'Rx'
+    end
+
+    properties (Hidden, Constant)
+        ComplexData = false
+    end
+
+    methods
+
+        %% Constructor
+        function obj = Rx(varargin)
+            obj = obj@matlabshared.libiio.base(varargin{:});
+            obj.enableExplicitPolling = false;
+            obj.EnabledChannels = 1;
+            obj.BufferTypeConversionEnable = true;
+            obj.uri = 'ip:analog.local';
+        end
+
+        function flush(obj)
+            flushBuffers(obj);
```

Comment:
Needs a docstring

Reply:
Still missing docstring

Reply:
When I see the diff, I see that docstring has already been added. Could you double check?